scarb/src/metadata/mod.rs

```diff
@@ -78,6 +81,14 @@ pub struct TargetMetadata {
     pub params: BTreeMap<String, Value>,
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+pub struct CompilationUnitMetadata {
+    pub package: PackageMetadata,
+    pub target: TargetMetadata,
+    pub components: Vec<PackageMetadata>,
```

Comment:
```suggestion
    pub package: PackageId,
    pub target: TargetMetadata,
    pub components: Vec<PackageId>,
```

I think, given that PackageMetadata for all packages is already printed, why duplicate this here?